pkg/engine/utils_test.go

```diff
@@ -1453,3 +1455,23 @@ func TestManagedPodResource(t *testing.T) {
 		assert.Equal(t, res, tc.expectedResult, "test %d/%s failed, expect %v, got %v", i+1, tc.name, tc.expectedResult, res)
 	}
 }
+
+func Test_checkKind(t *testing.T) {
+	match := checkKind([]string{"*"}, "Deployment", schema.GroupVersionKind{Kind: "Deployment", Group: "", Version: "v1"})
+	assert.Equal(t, match, true)
+
+	match = checkKind([]string{"Pod"}, "Pod", schema.GroupVersionKind{Kind: "Pod", Group: "", Version: "v1"})
+	assert.Equal(t, match, true)
+
+	match = checkKind([]string{"v1/Pod"}, "Pod", schema.GroupVersionKind{Kind: "Pod", Group: "", Version: "v1"})
+	assert.Equal(t, match, true)
+
+	match = checkKind([]string{"tekton.dev/v1beta1/TaskRun"}, "TaskRun", schema.GroupVersionKind{Kind: "TaskRun", Group: "tekton.dev", Version: "v1beta1"})
+	assert.Equal(t, match, true)
+
+	match = checkKind([]string{"tekton.dev/v1beta1/TaskRun/status"}, "TaskRun", schema.GroupVersionKind{Kind: "TaskRun", Group: "tekton.dev", Version: "v1beta1"})
+	assert.Equal(t, match, true)
+
+	match = checkKind([]string{"v1/pod.status"}, "Pod", schema.GroupVersionKind{Kind: "Pod", Group: "", Version: "v1"})
```

Comment:
Looks like both `/` and `.` are supported as of now. 

@JimBugwadia - we need to clarify how these are supported in the doc. Will you log an issue for tracking?

Reply:
https://github.com/kyverno/website/issues/533